packaging/nnstreamer.spec

```diff
@@ -285,22 +285,25 @@ meson --buildtype=plain --prefix=%{_prefix} --sysconfdir=%{_sysconfdir} --libdir
 
 ninja -C build %{?_smp_mflags}
 
-%if 0%{?unit_test}
     export NNSTREAMER_BUILD_ROOT_PATH=$(pwd)
     pushd build
     export GST_PLUGIN_PATH=$(pwd)/gst/nnstreamer
     export NNSTREAMER_CONF=$(pwd)/nnstreamer-test.ini
     export NNSTREAMER_FILTERS=$(pwd)/ext/nnstreamer/tensor_filter
     export NNSTREAMER_DECODERS=$(pwd)/ext/nnstreamer/tensor_decoder
+%if 0%{?enable_nnfw_r}
+    ./tests/tizen_nnfw_runtime/unittest_nnfw_runtime_raw --gst-plugin-path=. --gtest_output="xml:unittest_nnfw_runtime_raw.xml"
```

Comment:
Not bad. 
Note that the gbs command ia based on QEMU/ARM.  So, the "**gbs build**" of the CI system will be hung if the statement (e.g., unittest_nnfw_runtime_raw) runs more than 1~2 minutes. So we have to apply this trial after checking the execution time of the statement in the local PC carefully.

Reply:
Even in GBS/ARM, it taks less than 1 sec:
```
[  112s] + ./tests/tizen_nnfw_runtime/unittest_nnfw_runtime_raw --gst-plugin-path=. --gtest_output=xml:unittest_nnfw_runtime_raw.xml
[  112s] [==========] Running 5 tests from 1 test case.
[  112s] [----------] Global test environment set-up.
[  112s] [----------] 5 tests from nnstreamer_nnfw_runtime_raw_functions
[  112s] [ RUN      ] nnstreamer_nnfw_runtime_raw_functions.check_existence
[  112s] [       OK ] nnstreamer_nnfw_runtime_raw_functions.check_existence (24 ms)
[  112s] [ RUN      ] nnstreamer_nnfw_runtime_raw_functions.open_close_00_n
[  112s] Model file (null.nnfw) or its metadata is not valid (not regular).[       OK ] nnstreamer_nnfw_runtime_raw_functions.open_close_00_n (1 ms)
[  112s] [ RUN      ] nnstreamer_nnfw_runtime_raw_functions.open_close_01_n
[  112s] nnfw_close called without proper nnfw_opennnfw_close called without proper nnfw_open[       OK ] nnstreamer_nnfw_runtime_raw_functions.open_close_01_n (26 ms)
[  112s] [ RUN      ] nnstreamer_nnfw_runtime_raw_functions.get_dimension
[  112s] 
[  112s] ** (process:22154): CRITICAL **: nnfw_tensors_info_get: assertion 'pdata != NULL' failed
[  112s] 
[  112s] ** (process:22154): CRITICAL **: nnfw_tensors_info_get: assertion 'pdata != NULL' failed
[  112s] 
[  112s] ** (process:22154): CRITICAL **: nnfw_tensors_info_get: assertion 'info != NULL' failed
[  112s] 
[  112s] ** (process:22154): CRITICAL **: nnfw_tensors_info_get: assertion 'info != NULL' failed
[  112s] [       OK ] nnstreamer_nnfw_runtime_raw_functions.get_dimension (3 ms)
[  112s] [ RUN      ] nnstreamer_nnfw_runtime_raw_functions.invoke
[  112s] 
[  112s] ** (process:22154): CRITICAL **: nnfw_tensor_memory_set: assertion 'pdata != NULL' failed
[  112s] 
[  112s] ** (process:22154): CRITICAL **: nnfw_tensor_memory_set: assertion 'mem != NULL' failed
[  112s] 
[  112s] ** (process:22154): CRITICAL **: nnfw_tensor_memory_set: assertion 'mem != NULL' failed
[  112s] 
[  112s] ** (process:22154): CRITICAL **: nnfw_tensor_memory_set: assertion 'mem != NULL' failed
[  112s] [       OK ] nnstreamer_nnfw_runtime_raw_functions.invoke (4 ms)
[  112s] [----------] 5 tests from nnstreamer_nnfw_runtime_raw_functions (59 ms total)
[  112s] 
[  112s] [----------] Global test environment tear-down
[  112s] [==========] 5 tests from 1 test case ran. (61 ms total)
[  112s] [  PASSED  ] 5 tests.
```